system.js

```diff
@@ -0,0 +1,21 @@
+let randomIndexArray=[]
+        for (i=1; i<7; i++){
+            randomNum=Math.floor(Math.random()*45)
```

Comment:
```suggestion
        for (let i = 1; i < 7; i++) {
            randomNum = Math.floor(Math.random() * 45)
```

for문 안에서 i를 만들 때 let을 붙이는 게 정석이예요!
js에서는 선언자가 없어도 사용 가능하지만 스코프를 예측하기 힘들어지고 의도를 알기 힘들 거예요

Reply:
확인했습니다! 다음부터 사용할 일이 있을 때 유의해서 작성하겠습니다!